src/main/resources/config/liquibase/changelog/20170128182344_added_entity_Merchant.xml

```diff
@@ -71,8 +71,54 @@
 
             <!-- jhipster-needle-liquibase-add-column - Jhipster will add columns here, do not remove-->
         </createTable>
+        <createTable tableName="customer">
+            <column name="id" type="bigint" autoIncrement="${autoIncrement}">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="name" type="varchar(50)">
+                <constraints nullable="false" />
+            </column>
+
+            <column name="email" type="varchar(40)">
+                <constraints nullable="false" />
+            </column>
+
+            <column name="first_line_of_address" type="varchar(255)">
+                <constraints nullable="true" />
+            </column>
+
+            <column name="second_line_of_address" type="varchar(255)">
+                <constraints nullable="true" />
+            </column>
+
+            <column name="city" type="varchar(255)">
+                <constraints nullable="true" />
+            </column>
+
+            <column name="postcode" type="varchar(10)">
+                <constraints nullable="false" />
+            </column>
+
+            <column name="facebook" type="varchar(20)">
+                <constraints nullable="true" />
+            </column>
+
+            <column name="instagram" type="varchar(20)">
+                <constraints nullable="true" />
+            </column>
+
+            <column name="date_created" type="timestamp">
+                <constraints nullable="false" />
+            </column>
+
+            <column name="last_edited" type="timestamp">
+                <constraints nullable="false" />
+            </column>
+        </createTable>
         <dropDefaultValue tableName="merchant" columnName="date_created" columnDataType="datetime"/>
         <dropDefaultValue tableName="merchant" columnName="last_edited" columnDataType="datetime"/>
+        <dropDefaultValue tableName="customer" columnName="date_created" columnDataType="datetime"/>
+        <dropDefaultValue tableName="customer" columnName="last_edited" columnDataType="datetime"/>
 
     </changeSet>
 </databaseChangeLog>
```

Comment:
this isnt how the liquibase is supposed to work. You should add a new file in that directory with the changes.

Liquibse then applies the files in order on startup.